### ISSUE TITLE
GameSir G7 support

### ISSUE THREAD

**Issue by JohnTheCoolingFan** [paroj/xpad#232]
Hi, I  have a GameSir G7, probably an xbox one controller ("Wired controller for xbox" is written on the box), it works on windows but doesn't work on linux with xpad.

Here's info about it from lsusb: `ID 3537:1001 Guangzhou Chicken Run Network Technology Co., Ltd. GameSir-G7 Controller for Xbox`

**Comment by barni2000** [comment]
Hi @JohnTheCoolingFan
I think this gamepad compatible with https://github.com/medusalix/xone
They add the vendorId at january https://github.com/medusalix/xone/commit/24674076034e21d99e5e09f9b5fbbeaf21a5f8b4

**Comment by JohnTheCoolingFan** [comment]
Yes, iirc they added upon my request on discord and I've been using this driver since

**Comment by loloxploit** [comment]
thanks guys working on pop o.s here appreaite it!

**Comment by ArtisticOwl** [comment]
do you guys where I can get support for my wired g7 se for mac?

**Comment by IAmJoker47** [comment]
Im having this problem with my G7 SE. On Arch based distros or just Arch, the controller keeps disconnecting and reconnecting and the xone drivers don't seem to work as it need the "linux-headers". Any solution to this issue?

**Comment by JohnTheCoolingFan** [comment]
@IAmJoker47 what do you mean by "don't seem to work as it need the 'linux-headers'"? Is there some error installing the package or from the driver?

**Comment by IAmJoker47** [comment]
First of all on Arch, the controller keeps disconnecting and reconnecting.

![Image](https://github.com/user-attachments/assets/6568c7a6-7f8a-48e3-831c-baa8dc16703a)

I've just been able to solve this by installing xone from the AUR but it's still not working. The LED doesn't light up and I cant do anything with the controller. Tried restarting my pc a couple of times but still not working.

**Comment by JohnTheCoolingFan** [comment]
Can't help you much more, G7 SE is a different device. I recommend going to the xpadneo discord server, they have a channel about xone. Tell them about your issue, they might help you with it. There's also some messages before already asking about G7 SE, you can use channel search to find those, they have been answered. In each case there are different things involved.

**Comment by IAmJoker47** [comment]
I'm sorry for taking your time. I've finally been able to resolve my problem. Turns out I only needed to install the xone-dkms-git package. Now my controller works perfectly fine.

**Comment by IAmJoker47** [comment]
I kinda fixed it while i was in the middle of replying to you soooooo yeah. Couldn't show you the message even I wanted to. Anyway thank you for your time.
